tests/test_datetime.py

```diff
@@ -62,8 +62,11 @@ async def test_duration_01(self):
         ''', durs)
         self.assertEqual(list(durs_from_db), durs)
 
-    @unittest.expectedFailure
     async def test_relative_duration_01(self):
+        try:
+            self.con.query("SELECT <cal::relative_duration>'1y'")
+        except errors.InvalidReferenceError:
+            self.skipTest("feature not implemented")
```

Comment:
Is there a better way to skip this test only on the stable version and not nightly? Or rather only for the server versions that don't support it?

Reply:
Well, you can introspect the DB and check if the `cal::relative_duration` type is defined. If not - skip the test. We generally trust the introspection. Alternatively, you can query the server version with https://www.edgedb.com/docs/edgeql/funcops/sys#function::sys::get_version